src/cli/bootstrap/bootstrap.ts

```diff
@@ -165,17 +170,20 @@ async function createTokenDistribution() {
     console.log('Create token distribution');
     const totalSupply = 50000000000.0;
     const allocations = {
-        'ecosystm.tmy': 0.3 * totalSupply,
-        'team.tmy': 0.15 * totalSupply,
-        'coinsale.tmy': 0.15 * totalSupply,
-        'ops.tmy': 0.4 * totalSupply,
+        'ecosystm.tmy': { id: 1, percentage: 0.3, amount: 0.3 * totalSupply },
+        'team.tmy': { id: 2, percentage: 0.15, amount: 0.15 * totalSupply },
+        'coinsale.tmy': { id: 3, percentage: 0.15, amount: 0.15 * totalSupply },
+        'ops.tmy': { id: 4, percentage: 0.4, amount: 0.4 * totalSupply },
     };
 
-    for (const [account, amount] of Object.entries(allocations)) {
-        await tokenContract.transfer(
+    await vestngContract.updatedate('2024-03-19T00:00:00', '2024-04-20T00:00:00', signer);
```

Comment:
can you make the two dates into variables. the launch date needs to be 15 years in the future to prevent withdrawals